examples/igrantio/test-data.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
We'll get a formalized version of the fixtures data soon. I have an old note to go over the current fixtures with @lalc - you can see them here: https://github.com/GovStackWorkingGroup/bb-consent/blob/main/examples/mock/djangoapp/fixtures.json

I wouldn't spend any time porting them, yet, but I'll make sure to mention when they are ready.